vars/fioriOnCloudPlatformPipeline.groovy

```diff
@@ -43,7 +43,14 @@ void call(parameters = [:]) {
 
         stage('deploy') {
 
-            neoDeploy(parameters)
+            def mtaBuildCfg = parameters.script.commonPipelineEnvironment.getStepConfiguration('mtaBuild', '')
+
+            if((mtaBuildCfg.platform == 'NEO') || (mtaBuildCfg.buildTarget == 'NEO')) {
+                neoDeploy(parameters)
+            }
+            else {
```

Comment:
This means we fallback also to `CF` in case nothing was configured or something stupid was configured, e.g. in case `mtaBuildCfg.platform='FOO'`. I would like to suggest to only trigger a cfDeploy in case `CF` is explicitly configured. In case we do not trigger a `neoDeploy` and also no `cfDeploy` we should have a log message, or even better: throw an exception.

Reply:
if the platform is neither NEO nor CF the build should fail and we never reach here. One exception is XS, but this one, i ignored.
if no platform is defined, NEO is the default by default.yaml.

well, i haven't invented the code, i guess it is intended to be as simple as possible since there is a cut and paste comment inside.

Reply:
done